lib/elixir_search_extractor_worker/csv_keywords_processing_worker.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ElixirSearchExtractor.ElixirSearchExtractorWorker.CsvKeywordsProcessingWorker do
```

Comment:
Maybe

```suggestion
defmodule ElixirSearchExtractorWorker.CsvKeywordsProcessingWorker do
```

We can create an empty `ElixirSearchExtractorWorker` module in the lib -> `lib/elixir_search_extractor_worker.ex`

Reply:
I'm thinking about the name maybe we can call `KeywordsProcessing` -> `KeywordFileProcessing` what do you think? or base we can call it `KeywordParseWorker` 💭